packages/common/src/utils/commandHandler.ts

```diff
@@ -1,11 +1,26 @@
-import { getComponent, addComponent } from '../../../engine/src/ecs/functions/EntityFunctions'
+import { getComponent, addComponent, hasComponent } from '../../../engine/src/ecs/functions/EntityFunctions'
```

Comment:
common folder cannot import from engine, as engine depends on common. please move all of these to the engine.